src/packages/@ncigdc/modern_components/GeneExpression/inchlib/index.js

```diff
@@ -230,8 +236,88 @@ import Color from 'color';
     self.min_size_draw_values = 20;
     self.column_metadata_row_height = self.min_size_draw_values;
 
+    // control hover color with opacity
+    self.hover_fill = '#3a3a3a';
+    self.hover_opacity_off = 0.7;
+    self.hover_opacity_on = 1;
+
+    // column metadata colors & legend info
+    self.MAX_DAYS_TO_DEATH = 3379;
+    self.MAX_AGE_AT_DIAGNOSIS = 90;
+    self.invalid_column_metadata_color = '#fff';
+    self.age_dx_colors = {
+      hue: 106,
+      max_light: 88,
+      min_light: 45,
+      sat: 25,
+    };
+
+    self.get_days_to_death_color = val => {
+      const red_green = Math.floor(255 - (val / self.MAX_DAYS_TO_DEATH * 255));
```

Comment:
Any chance we can have a comment explaining the logic of this equation?

Reply:
sure thing!

Reply:
does this work?

```js
self.get_days_to_death_color = val => {
      // create red & green values for RGB().
      // will result in a shade of blue.
      // higher value = darker blue.
      const red_green = Math.floor(255 - (val / self.MAX_DAYS_TO_DEATH * 255));
      return isNaN(red_green) // i.e. val is "not reported"
        ? self.invalid_column_metadata_color
        : `rgb(${red_green},${red_green},255)`;
    };

    self.get_age_at_diagnosis_color = val => {
      // create lightness value for HSL().
      // will result in a shade of green.
      // higher value = darker green.
      const percentage = 1 - (val / self.MAX_AGE_AT_DIAGNOSIS);
      const lightness = (percentage * (self.age_dx_colors.max_light - self.age_dx_colors.min_light)) + self.age_dx_colors.min_light;
      return isNaN(percentage) // i.e. val is "not reported"
        ? self.invalid_column_metadata_color
        : `hsl(${self.age_dx_colors.hue},${self.age_dx_colors.sat}%,${lightness}%)`;
    }
```